README.md

```diff
@@ -20,4 +20,4 @@ Add your answers to this markdown file.
 
 2. The time compelexity could only be expected to be log(10000) if the tree is balanced, but there is no way to know the exact time it would take to find the same element. It would depend on many different factors such as hardware and the data that was put into the tree. 
 
-3. One reason this could have happened is because the tree is unbalanced, which causes the computer to do more work in search of the element. Another reason may be that the size of the array is too big for one memory allocation, so the computer has to use a larger but slower memory unit to store the tree. A third reason this could happen is because the search function for the element in the tree is written enefficiently and calls unnecessary function calls or iterations. 
+3. One reason this could have happened is because the hardware of the computer executing the code may handle the size of this tree poorly. Another reason may be that the size of the data that was added to the new, larger tree, is larger than the data in the smaller tree which takes the computer a lot longer to process. A third reason this could happen is because the search function for the element in the tree is written enefficiently and calls unnecessary function calls or iterations.
```

Comment:
Sorry I was just updating what I had. If the hard drive copied the tree at every step of recursion, that would increase the space exponentially at every step right?

Reply:
No. The tree height only grows logarithmic.

Reply:
The tree is of size n though right? So if it copied the tree at each step the space would increase by n^i where i is the number of times you copy it. I'm not sure if I am going off track with this.

Reply:
Why would the tree be copied at each step? And what is a "step" here?

Reply:
I'm not sure I am just trying to think of any possibility of what could be causing the functions runtime to be so slow. By each step I am meaning at each step of recursion. I will just try to do more research and see if I can find something else that could exponentially affect the runtime.

Reply:
One of the reasons that came up in class today was the case that the computer may have to switch from RAM to a disk memory because of the input size. I had this idea in my original commit on this pull request but you said for both runs it would have to fetch from main memory. I was wondering if maybe there was some confusion in the way I worded it.

Reply:
As I've said, you would need to be clear that this is something that's *different* between the two runs.